wpilibj/src/main/java/edu/wpi/first/wpilibj/simulation/ElevatorSim.java

```diff
@@ -105,6 +105,74 @@ public ElevatorSim(
         null);
   }
 
+  /**
+   * Creates a simulated elevator mechanism.
+   *
+   * @param kG The gravity gain.
+   * @param gearbox The type of and number of motors in the elevator gearbox.
+   * @param gearing The gearing of the elevator (numbers greater than 1 represent reductions).
+   * @param drumRadiusMeters The radius of the drum that the elevator spool is wrapped around.
+   * @param minHeightMeters The min allowable height of the elevator.
+   * @param maxHeightMeters The max allowable height of the elevator.
+   * @param simulateGravity Whether gravity should be simulated or not.
+   * @param startingHeightMeters The starting height of the elevator.
+   */
+  public ElevatorSim(
+      double kG,
+      DCMotor gearbox,
```

Comment:
You forgot Kv and Ka.

Reply:
Do I need to remove KG?

Reply:
I'm a little lost.  Should kV, kA, and kG be capable of generating the gearbox, gearing, and drumRadius fields that are required?